E621Client/Area/Iqdb/E621Client.cs

```diff
@@ -21,44 +21,17 @@ public partial class E621Client
         // an IQDB query seems to prevent 429 responses.
         private const int IqdbRequestInterval = 0;
         private const int IqdbRequestDelay = 2000;
+        
+        /// <inheritdoc cref="IE621Client.IqdbAllowedFormats"/>
+        public IReadOnlyDictionary<string, string[]> IqdbAllowedFormats { get; }
```

Comment:
`IqdbAllowedFormats` is defined in `E621Constants`, right? So this property is never used

Reply:
True i forgot to remove them